composer.json

```diff
@@ -22,9 +22,14 @@
     "require": {
         "php": ">=5.3.2",
         "symfony/doctrine-bridge": "~2.1",
+        "symfony/console": "~2.1",
         "doctrine/doctrine-bundle": "~1.0",
+        "doctrine/orm": "~2.0",
```

Comment:
FYI: This code will not work without ORM as it has a hard dependency on Doctrine\ORM\EntityManagerInterface in \Doctrine\Common\DataFixtures\Purger\ORMPurger.
But this is up to you :)